Jenkinsfiles/utils.groovy

```diff
@@ -2,10 +2,38 @@
  * Define utility functions.
  */
 
-def checkout_github(repo, branch, relative_target_dir) {
+def get_commit_tag() {
+    return env.GIT_COMMIT.take(7)
```

Comment:
whats does the value 7 correspond to?

Reply:
It means "take the first 7 characters" of the git commit hash (so we match the tag we use for quay.io).

Reply:
ah, ok, I'm not used to the Groovy APIs.